kubernetes/kserve/Dockerfile.dev

```diff
@@ -28,21 +28,20 @@ RUN --mount=type=cache,id=apt-dev,target=/var/cache/apt \
   dpkg-dev \
   g++ \
   git \
-  python3-dev \
+  python3.8-dev \
+  python3-pip \
   build-essential \
   openjdk-11-jdk \
   curl \
   vim \
   && rm -rf /var/lib/apt/lists/* \
   && cd /tmp \
-  && curl -O https://bootstrap.pypa.io/get-pip.py \
-  && python3 get-pip.py
 
-RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1 \
-  && update-alternatives --install /usr/local/bin/pip pip /usr/local/bin/pip3 1
+RUN update-alternatives --remove python /usr/bin/python \
+  && update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1
```

Comment:
Same here. Why specific version python3.8 is needed?

Reply:
The base image `ubuntu-18.04` comes with python 3.6. There were two reasons why we wanted to move into new version

We are getting the following error while building the image using python 3.6
```
Traceback (most recent call last):
  File "/home/model-server/kserve_wrapper/__main__.py", line 4, in <module>
    from importlib.metadata import version
ModuleNotFoundError: No module named 'importlib.metadata'
```

Had a discussion with @chauhang and changed it to 3.8 , as 3.6 is bit old and all our kserve testing were done using python 3.8.